atspi-common/src/object_match.rs

```diff
@@ -61,13 +61,16 @@ pub struct ObjectMatchRule {
 /// <https://gitlab.gnome.org/federico/at-spi2-core/-/commit/4885efedeef71e0df8210622771a0b1bb10e194d>
 impl Type for ObjectMatchRule {
 	const SIGNATURE: &'static Signature = &Signature::static_structure(&[
-		<Vec<i32>>::SIGNATURE,
```

Comment:
Why do you prefer the explicit typing here? I specifically used the associated const `SIGNATURE` of other types because it was more readable.